README.md

```diff
@@ -2,20 +2,31 @@
 
 Starter project for a chess-based TDD exercise.
 
-## Running the application
+## Getting the application up and running
 
-To run the application, first ensure that you have installed Poetry on your system. Install
-dependencies using the command ``poetry install``.
+### Installing dependencies
+First, let's ensure that you have installed python and pip on your system. 
 
-To run the application, use the command ``poetry run flask run`` and navigate to http://localhost:5000. You can reset the board at any time by restarting the server.
+- Run `python --version` and you should see a version number. If not, you may need to install Python on your machine. On ACG, this should be already available - ensure you're using the image specified in the instructions.
+- Run `pip --version` and you should also see a version number. If not, you may need to install pip on your machine. On ACG, you can do `sudo apt install python3-pip` to install pip.
+
+After that, install dependencies using the command `pip install -r requirements.txt`.
```

Comment:
Do we want to pass over virtual environments entirely, is there any risk that we cause issues with work dependencies? I'm inclined to say it's fine - if a learner is using Python at work then they should already be aware 🤞

Reply:
I was originally imagining the ACG instance which is ephemeral enough that it doesn't matter, but I had neglected the local machine scenario. We could provide venv instructions, do you think it's worth doing? We could halfway house it and say "if you're working locally you may wish to use a venv if you know how"?

Reply:
Yeah that sounds like a nice halfway 😄